### PR TITLE
Removed redundant text-transform instances

### PR DESCRIPTION
**Pull request by PaarthAgarwal**
This PR fixes "Refactor Code" and "Configure Tailwind" part of #8121.

- I have removed all redundant text-transform instances from the stylesheets.
- Disabled the text-transform Tailwind plugin.

### CODE REVIEW

File: wagtail/admin/static_src/wagtailadmin/css/normalize.css [hunk 1]
```diff
@@ -421,18 +421,7 @@ input {
...
 }
 
-/**
- * Address inconsistent `text-transform` inheritance for `button` and `select`.
- * All other form control elements do not inherit `text-transform` values.
- * Correct `button` style inheritance in Chrome, Safari 5+, and IE 6+.
- * Correct `select` style inheritance in Firefox 4+ and Opera.
- */
-
 button,
```
Comment (lb-) [inline wagtail/admin/static_src/wagtailadmin/css/normalize.css:424]: pretty sure this line can be removed from what I can see

**Review by lb- — CHANGES_REQUESTED** [review]
I think there is one minor typo, looks good otherwise.

**Comment by PaarthAgarwal** [comment]
> I think there is one minor typo, looks good otherwise.

Can you point it out?

File: wagtail/admin/static_src/wagtailadmin/css/normalize.css (hunk 1 above)
Comment (PaarthAgarwal) [reply wagtail/admin/static_src/wagtailadmin/css/normalize.css:424]: sure

**Comment by PaarthAgarwal** [comment]
I have fixed the typo as mentioned.

**Comment by PaarthAgarwal** [comment]
@lb- have a look

**Review by lb- — APPROVED** [review]
looking great @PaarthAgarwal  thank you!

**Comment by lb-** [comment]
Did not add this to changelog as already included in there (related to text transform)

**Pull request merged**